src/org/usfirst/frc2876/PowerUp2018/OI.java

```diff
@@ -135,5 +145,44 @@ public double getRightTrigger() {
 			A_BUTTON=1,
 			B_BUTTON=2;
 
+	/**

```

Comment:
Did you pull this smartdashboard code from somewhere else?

Reply:
Yes, I did. I found it on a chief delphi thread.